src/main/webapp/user-page.html

```diff
@@ -34,22 +32,60 @@ <h1 id="page-title">User Page</h1>
     <div id="about-me-container">Loading...</div>
       <div id="about-me-form" class="hidden">
       <form action="/about" method="POST">
-        <textarea name="about-me" placeholder="about me" rows=4 required></textarea>
+        <textarea name="about-me" id="about-me-box" placeholder="about me" rows=4 required></textarea>
         <br/>
         <input type="submit" value="Submit">
+        <script>
+          ClassicEditor.create( document.getElementById('about-me-box'), {
+            removePlugins: [ 'blockQuote' ],
+            toolbar: [ 'heading', '|', 'bold', 'italic', 'link', 'bulletedList', 'numberedList'],
+            heading: {
+              options: [
+                { model: 'paragraph', title: 'Paragraph', class: 'ck-heading_paragraph' },
+                { model: 'heading1', view: 'h1', title: 'Title', class: 'ck-heading_heading1' },
+                { model: 'heading2', view: 'h2', title: 'Heading', class: 'ck-heading_heading2' }
+              ]
+            }
+          } )
+          .then( editor => {
+            console.log( editor );
+          } )
+          .catch( error => {
+            console.log( error );
+          } );
+        </script> 
       </form>
     </div>
-
+    
     <form id="message-form" action="/messages" method="POST" class="hidden">
       Enter a new message:
       <br/>
       <textarea name="text" id="message-input"></textarea>
       <br/>
-      <input type="submit" value="Submit">
+      <input type="submit" value="Submit"> 
+      <script>
+        ClassicEditor
+          .create( document.getElementById('message-input'), {
+            removePlugins: [ 'blockQuote' ],
+            toolbar: [ 'heading', '|', 'bold', 'italic', 'link', 'bulletedList', 'numberedList'],
+            heading: {
+              options: [
+                { model: 'paragraph', title: 'Paragraph', class: 'ck-heading_paragraph' },
+                { model: 'heading1', view: 'h1', title: 'Title', class: 'ck-heading_heading1' },
+                { model: 'heading2', view: 'h2', title: 'Heading', class: 'ck-heading_heading2' }
+              ]
+            }
+          } )
+	  .then( editor => {
```

Comment:
I think this can be simplified to .then( editor => console.log(editor);}). I don't think the extra brackets are necessary. Can you check? Fix throughout file if it can be simplified.